aggregator/src/cache.rs

```diff
@@ -16,11 +16,15 @@ use std::{
 use tokio::{spawn, task::JoinHandle, time::sleep};
 use tracing::error;
 
+pub const DEFAULT_REFRESH_INTERVAL: StdDuration =
+    StdDuration::from_secs(60 * 30 /* 30 minutes */);
+
```

Comment:
This move is to avoid having to qualify references to this const with a turbofish and fake `Clock`.